infrastructure/sprod.tfvars

```diff
@@ -1,4 +1,4 @@
 capacity = "2"
-idam_api_url = "https://preprod-idamapi.reform.hmcts.net:3511"
+idam_api_url = "http://betadevbccidamapplb.reform.hmcts.net"
```

Comment:
it's not even https?

Reply:
https doesn't work anyways it would be soon replaced with strategic solution so should be fine for now.